.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Scope
+
+
+## To Reproduce
+To reproduce this issue : 
+
+* ...
+* ..
```

Comment:
Remove these - going to be annoying removing them every time an issue is opened.